src/scss/common/typography.scss

```diff
@@ -54,7 +54,7 @@ h4, .h4 {
 
 a {
 	display: inline-block;
-	transition: color ease-out $tr;
+	transition: all ease-out $tr;
```

Comment:
Don't use 'all' in transition. If you need to animate several properties - use them.